sound/soc/sof/Kconfig

```diff
@@ -54,6 +54,7 @@ config SND_SOC_SOF_COMPRESS
 
 config SND_SOC_SOF_DEBUG_PROBES
 	tristate
+	depends on SND_SOC_SOF
```

Comment:
This can only be selected, it can not switch on just like that in config, how did you managed to have a config like that?
It is only selected by Intel HDA.

Reply:
Oh, indeed it work. I was worried that since SND_SOC_SOF_DEBUG_PROBES is selected by SND_SOC_SOF_HDA_PROBES - which is boolean - that selecting m/y would not work correctly trough SND_SOC_SOF_HDA_COMMON -> SND_SOC_SOF_HDA_PROBES -> SND_SOC_SOF_DEBUG_PROBES. I have no idea what force on earth turns SND_SOC_SOF_DEBUG_PROBES to m when I switch SND_SOC_SOF_TIGERLAKE to m from y, but somehow it works. Somehow the selection over boolean option retains the original m/y state.